<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelName</class>
 <widget class="QDialog" name="LevelName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Level Name</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59);</string>
  </property>
  <widget class="MovableQFrame" name="mfra_TitleBarDialog">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(44, 44, 44);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_DialogTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(44, 44, 44);
color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>BXGameEngine Level Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CloseDialog">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(44, 44, 44);
color: rgb(0, 170, 255);
border: none;
}
QPushButton::hover {
	background-color: rgb(0, 132, 198);
	color: rgb(44, 44, 44);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MinimizeDialog">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(44, 44, 44);
color: rgb(0, 170, 255);
border: none;
}
QPushButton::hover {
	background-color: rgb(0, 132, 198);
	color: rgb(44, 44, 44);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_Ok">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_inputLevelName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
   <property name="placeholderText">
    <string>NewLevelName</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Notification">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>184</y>
     <width>392</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MovableQFrame</class>
   <extends>QFrame</extends>
   <header>../Headerfiles/Qt/MovableQFrame.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_inputLevelName</tabstop>
  <tabstop>btn_Ok</tabstop>
  <tabstop>btn_Cancel</tabstop>
  <tabstop>btn_MinimizeDialog</tabstop>
  <tabstop>btn_CloseDialog</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_CloseDialog</sender>
   <signal>clicked()</signal>
   <receiver>LevelName</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Ok</sender>
   <signal>clicked()</signal>
   <receiver>LevelName</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>LevelName</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_MinimizeDialog</sender>
   <signal>clicked()</signal>
   <receiver>LevelName</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok()</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
